subiquity/core.py

```diff
@@ -98,5 +111,12 @@ def unhandled_input(self, key):
         if key == 'f1':
             if not self.showing_help:
                 self.ui.right_icon.open_pop_up()
+        elif key == 'ctrl z':
+            self.debug_shell()
         else:
             super().unhandled_input(key)
+
+    def debug_shell(self):
+        self.run_command_in_foreground(
+            "clear && echo {} && bash".format(shlex.quote(DEBUG_SHELL_INTRO)),
+            shell=True)
```

Comment:
Note to self, this pipe runs in `/bin/sh` to exec bash.

Reply:
Printing the message this way is a touch gross, we could also modify run_command_in_foreground to take a hook to run after we have disconnected from stdin or something. This works though.